includes/WebRequest.php

```diff
@@ -1134,6 +1134,23 @@ public function isValidWriteRequest( \User $user ) {
 	public function isWikiaInternalRequest() {
 		return $this->getHeader( self::WIKIA_INTERNAL_REQUEST_HEADER ) !== false;
 	}
+
```

Comment:
Can you please add begin/end comments denoting that this is a Wikia change? This file is originally from MediaWiki, so conventionally, within MW files, in-house changes are marked. Here is an [Example](https://github.com/Wikia/app/blob/master/includes/WebRequest.php#L1098-L1106).